src/index.js

```diff
@@ -0,0 +1,162 @@
+'use strict';
+
+function Modal(clickElm, modalParent, options) {
+  options = options || {};
+
+  var that = this;
+  clickElm.addEventListener('click', function(e) {
+    e.preventDefault();
+    that.show();
+  }, false);
+  this.clickElm = clickElm;
+  this.modalParent = modalParent || document.documentElement;
```

Comment:
hmm, yeah should think about that a bit more. I'd assumed multiple instances, so we could just look for other .fxa-pay-containers that don't have an id matching the uniquely generated instance id and nuke them.